L/libcoreblas/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcoreblas"
```

Comment:
Naming nit: we should follow the name of the upstream project, not that of the product: https://docs.binarybuilder.org/stable/building/#Name
```suggestion
name = "CoreBlas"
```
Note that the name of the directory should match this name, so this file should be moved to `C/CoreBlas/build_tarballs.jl`